File: apps/frontend/mobile/src/types/Club.ts
export interface Club {
    id: string;
    rawName: string;
    name: string;
    address: string | null;
    city: string | null;
    postalCode: string | null;
    email: string | null;
    phoneNumber: string | null;
    website: string | null;
    logoUrl: string | null;
    active: boolean;
    latitude: number | null;
    longitude: number | null;
    createdAt: string | null;
    lastUpdate: string | null;
}

export interface ClubSearchDocDTO {
    id: string;
    name: string;
    logoUrl: string | null;
    city: string;
}

File: apps/backend/club-scraper/models/club.py
from dataclasses import dataclass, field
from datetime import datetime
from typing import Optional

@dataclass
class Club:
    id: str
    rawName: str
    name: str
    city: Optional[str] = None
    postalCode: Optional[str] = None
    address: Optional[str] = None
    email: Optional[str] = None
    phoneNumber: Optional[str] = None
    website: Optional[str] = None
    logoUrl: Optional[str] = None
    active: bool = True
    createdAt: Optional[datetime] = None
    lastUpdate: Optional[datetime] = None